Doc/source/optional.rst

```diff
@@ -1,142 +1,112 @@
-:orphan:
-
 Optional Dependencies
 =====================
 
-The fonttools PyPI distribution also supports so-called "extras", i.e. a
-set of keywords that describe a group of additional dependencies, which can be
-used when installing via pip, or when specifying a requirement.
-For example:
+The FontTools PyPI distribution supports "extras", which are groups of additional dependencies that can be installed alongside FontTools via pip to enable specific features. For example:
 
 .. code:: sh
 
     pip install fonttools[ufo,lxml,woff,unicode]
 
-This command will install fonttools, as well as the optional dependencies that
-are required to unlock the extra features named "ufo", etc.
-
-.. note::
-
-    Optional dependencies are detailed by module in the list below with the ``Extra`` setting that automates ``pip`` dependency installation when this is supported.
+This command installs FontTools along with the optional dependencies required for features like "ufo", "lxml", "woff", and "unicode".
 
+.. note:: sh
 
+    Optional dependencies are listed by module below, along with the `Extra` setting that automates `pip` dependency installation when supported.
 
 :py:mod:`fontTools.misc.etree`
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+--------------------
 
-The module exports a ElementTree-like API for reading/writing XML files, and allows to use as the backend either the built-in ``xml.etree`` module or `lxml <https://lxml.de>`__. The latter is preferred whenever present, as it is generally faster and more secure.
+This module provides an ElementTree-like API for reading and writing XML files. It supports two backend options:
 
-*Extra:* ``lxml``
+- Built-in `xml.etree` module
+- `lxml` preferred for improved speed and security.
 
+*Extra:* `lxml`
 
 :py:mod:`fontTools.ufoLib`
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Package for reading and writing UFO source files; it requires:
+----------------
 
-* `fs <https://pypi.org/pypi/fs>`__: (aka ``pyfilesystem2``) filesystem abstraction layer.
+Package for reading and writing UFO source files. Dependencies include:
```

Comment:
My preference would be to spell out "Unified Font Object (UFO)" here, on first usage at this entry point. It's clear if the reader already knows what UFO is, but you are always going to get people who are diving in without a lot of prior background reading, because they've got some ultra-specific problem they're trying to fix.

Reply:
I should maybe add that I don't think it's necessary to *always* spell out UFO and other abbreviations, but this is intro material, dealing with step-one installation stuff, so I think the need is higher here....